README.md

```diff
@@ -6,82 +6,82 @@ OpenShift Application Platform
 [![Travis](https://travis-ci.org/openshift/origin.svg?branch=master)](https://travis-ci.org/openshift/origin)
 [![Jenkins](https://ci.openshift.redhat.com/jenkins/buildStatus/icon?job=devenv_ami)](https://ci.openshift.redhat.com/jenkins/job/devenv_ami/)
 [![Join the chat at freenode:openshift-dev](https://img.shields.io/badge/irc-freenode%3A%20%23openshift--dev-blue.svg)](http://webchat.freenode.net/?channels=%23openshift-dev)
+[![Licensed under Apache License version 2.0](https://img.shields.io/github/license/openshift/origin.svg?maxAge=2592000)](https://www.apache.org/licenses/LICENSE-2.0)
 
-OpenShift Origin is a distribution of [Kubernetes](https://kubernetes.io) optimized for continuous application development and multi-tenant deployment.  Origin adds developer and operations-centric tools on top of Kubernetes to enable rapid application development, easy deployment and scaling, and long-term lifecycle maintenance for small and large teams.
+***OpenShift Origin*** is a distribution of [Kubernetes](https://kubernetes.io) optimized for continuous application development and multi-tenant deployment.  Origin enables teams of all sizes to quickly develop, deploy, and scale applications, reducing the length of development cycles and operational effort.
 
 [![Watch the full asciicast](docs/openshift-intro.gif)](https://asciinema.org/a/49402)
 
 **Features:**
 
-* Easily build applications with integrated service discovery, DNS, load balancing, failover, health checking, persistent storage, and fast scaling
-* Push source code to your Git repository and automatically roll out containerized applications (micro or macro)
-* Easy to use client and web console for building and monitoring applications
-* Centralized administration and management of an entire stack, team, or organization
-  * Templatize the components of your system, reuse them, and iteratively deploy them over time
-  * Roll out changes to software stacks to your entire organization in a controlled fashion
-* Team and user isolation of containers, builds, and network communication in an easy multi-tenant system
-  * Allow developers to run containers securely with fine-grained controls in production
-  * Limit, track, and manage the developers and teams on the platform
-* Integrated Docker registry, automatic edge load balancing, cluster logging, and integrated metrics
+**For developers:**
```

Comment:
There is no longer a clean separation between the personas, especially in
the audience our open source community reaches.  I don't mind ordering
giving a subtle nod, but I don't see a strong distinction in the people who
care about this readme.

On Jun 20, 2016, at 7:23 PM, Jonathan Yu <notifications@github.com> wrote:

In README.md
<https://github.com/openshift/origin/pull/8007#discussion_r67784876>:

>
>  [![Watch the full asciicast](docs/openshift-intro.gif)](https://asciinema.org/a/49402)
>
>  **Features:**
>
> -* Easily build applications with integrated service discovery, DNS, load balancing, failover, health checking, persistent storage, and fast scaling
> -* Push source code to your Git repository and automatically roll out containerized applications (micro or macro)
> -* Easy to use client and web console for building and monitoring applications
> -* Centralized administration and management of an entire stack, team, or organization
> -  * Templatize the components of your system, reuse them, and iteratively deploy them over time
> -  * Roll out changes to software stacks to your entire organization in a controlled fashion
> -* Team and user isolation of containers, builds, and network communication in an easy multi-tenant system
> -  * Allow developers to run containers securely with fine-grained controls in production
> -  * Limit, track, and manage the developers and teams on the platform
> -* Integrated Docker registry, automatic edge load balancing, cluster logging, and integrated metrics
> +**For developers:**

@danmcp <https://github.com/danmcp> I guess my concern is that just listing
a long list of features will hurt readability and make the file less
approachable. The structure is meant to imply "as a (Developer | Operator),
you're probably most interested in X, Y, Z" but doesn't mean that both
groups shouldn't read through everything. It lends a little context as to
why the features are worthwhile, and for a DevOps person, informs which
"hat" you should wear while reading through each point.

If you feel strongly about this, I will merge the two sections together. I
feel that we're doing our users a disservice by just providing a laundry
list of features without connection to the specific personas that they
target.

—
You are receiving this because you were assigned.
Reply to this email directly, view it on GitHub
<https://github.com/openshift/origin/pull/8007/files/bc4da8fa99b5d01b492ccd85bada33fcb9cd33dd#r67784876>,
or mute the thread
<https://github.com/notifications/unsubscribe/ABG_pxp2_deWhVk7HVTboy9vVfMRfLGoks5qNyD8gaJpZM4HwhAG>
.